app/src/server/api/external/v1Api.router.ts

```diff
@@ -334,6 +301,149 @@ export const v1ApiRouter = createOpenApiRouter({
 
       return { status: "ok" };
     }),
+  updateLogTags: openApiProtectedProc
+    .meta({
+      openapi: {
+        method: "POST",
+        path: "/update-log-tags",
```

Comment:
Perhaps "/logs/update-tags" might be the better route here?

Reply:
No strong opinion on my part, I'll update it.